### DISCUSSION TITLE
How can I use four cameras at once?

### DISCUSSION THREAD

**Discussion by jocelynasp** [bonsai-rx/bonsai#878] [Q&A] [answered]
My goal is to record from four or more cameras at once. These cameras are going to be recording mice behavior from 4 different behavior boxes (MEDPC) and for some experiments there will be more (when we use reward ports, we want to see which mouse gets the reward). Before I do any tracking or anything my number one goal is to get those four cameras running at the same time. 

I'm very new to Bonsai, so the only thing I have so far is recording from one ca
<img width="576" alt="CameraCapture Workflow" src="https://user-images.githubusercontent.com/99834393/164090444-e4ddd5ce-ff5e-4fa4-865b-26898fee1696.png">
mera.

**Comment by bruno-f-cruz** [comment, marked as answer]
Hey,
Using 'CameraCapture', you can access other cameras by chaging the 'Index' property.
![image](https://user-images.githubusercontent.com/7049351/164204624-fc555a55-c7ad-4ac1-b660-a8ad19148959.png)
Similarly, if you use 'VideoCaptureDevice', you can also change this property to achieve the same outcome:
![image](https://user-images.githubusercontent.com/7049351/164204901-8c3db1c3-1c54-4923-a1ff-c3122968fc15.png)

**Reply by jocelynasp** [reply]
I have done that, but I don't understand how I can have four cameras running at the same time in the same workflow. Is that possible? The idea is that I'd like to start Bonsai and have them all turn on and start recording all at once. What nodes should I be using to create that workflow (if it's possible).

**Reply by bruno-f-cruz** [reply]
You can add multiple 'CameraCapture' nodes to the same workflow. As long as they have a different 'Index' property it will not be a problem.
